app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java

```diff
@@ -25,17 +32,25 @@
 @PersistenceExceptionLivenessApply
 public class PersistenceExceptionLivenessInterceptor {
 
-//    private static final Logger log = LoggerFactory.getLogger(PersistenceExceptionLivenessInterceptor.class);
-
-    private static final Set<Class<? extends Exception>> IGNORED = new HashSet<>();
+    private static final Set<Class<? extends Throwable>> IGNORED = new HashSet<>();
 
     static {
+        // Inherit from io.apicurio.registry.rest.RegistryExceptionMapper
+        // to avoid manual sync, even though some exceptions may be irrelevant
+        for (Map.Entry<Class<? extends Throwable>, Integer> entry : RegistryExceptionMapper.getCodeMap().entrySet()) {
+            if(entry.getValue() != HTTP_INTERNAL_ERROR) // Sanity check
+                IGNORED.add(entry.getKey());
+        }
+        // Additional ignored exceptions (may be already included)
         IGNORED.add(AlreadyExistsException.class);
         IGNORED.add(ArtifactAlreadyExistsException.class);
         IGNORED.add(ArtifactNotFoundException.class);
+        IGNORED.add(InvalidArtifactStateException.class);
         IGNORED.add(NotFoundException.class);
+        IGNORED.add(RegistryStorageException.class);
```

Comment:
This one is actually internal error ... and as such can be left for the mapper to handle it.